web/js/layers/wv.layers.modal.js

```diff
@@ -367,7 +367,7 @@ wv.layers.modal = wv.layers.modal || function(models, ui, config) {
         redoScrollbar();
         $selectedCategory.iCheck({checkboxClass: 'icheckbox_square-red'});
         $breadcrumb.show();
-        
+
     };
 
     var drawAllLayers = function() {
```

Comment:
I have worked on breaking up most of the methods in this file and documenting with jsdoc. drawMeasurements is tough one to break up further since the HTML for this view is built by finding the current category, looping through it, building the measurement html, appending that, finding the nested sources, appending that, and finding the descriptions and appending that to build the full view.